src/Core/Components/Avatar/FluentAvatar.razor

```diff
@@ -0,0 +1,35 @@
+﻿@namespace Microsoft.FluentUI.AspNetCore.Components
+@using Microsoft.AspNetCore.Components.Rendering;
+@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@inherits FluentComponentBase
+
+<fluent-avatar id="@Id"
+               role="button"
+               tabindex="0"
+               class="@ClassValue"
+               style="@StyleValue"
+               active="@GetActiveValue"
+               appearance="@ActiveAppearance.ToAttributeValue()"
+               color="@Color.ToAttributeValue()"
+               initials="@GetInitialsValue"
+               name="@GetNameValue"
+               shape="@Shape.ToAttributeValue()"
+               size="@Size.ToAttributeValue()"
+               @onclick="@OnClickHandlerAsync"
+               @onkeydown="@OnKeyDownHandlerAsync"
```

Comment:
What is the goal of this `onkeydown` method? I don't think we need that. Only `OnClick` is enought.